docs/src/main/sphinx/release/release-461.md

```diff
@@ -0,0 +1,100 @@
+# Release 461 (9 Oct 2024)
+
+## General
+
+* Rename the configuration property `max-tasks-waiting-for-execution-per-stage`
+  to `max-tasks-waiting-for-execution-per-query` and the session property
+  `max_tasks_waiting_for_node_per_stage` to
+  `max_tasks_waiting_for_node_per_query` to match implemented semantics.
+  ({issue}`23585`)
+
+## Security
+
+* Add support for BCrypt 2A, 2B, 2X versions. ({issue}`23648`)
+
+## Web UI
+
+## JDBC driver
+
+## Server RPM
+
+## Docker image
+
+## CLI
+
+## Accumulo connector
+
+## BigQuery connector
+
+## Blackhole connector
+
+## Cassandra connector
+
+## ClickHouse connector
+
+## Delta Lake connector
+
+## Druid connector
+
+## Elasticsearch connector
+
+## Exasol connector
+
+## Google Sheets connector
+
+## Hive connector
+
+* Add support for using the Hive `CombineTextInputFormat` in storage descriptors. ({issue}`21842`)
```

Comment:
@electrum and @hashhar is that RN note okay?

Reply:
Maybe leave out the `in storage descriptors`.

Or what about: ```Add support for reading Hive tables that use `CombineTextInputFormat` ```?

Reply:
Thats good .. I will update to that